spyder/config/main.py

```diff
@@ -300,15 +300,13 @@
               }),
             ('workingdir',
              {
-              'editor/open/browse_scriptdir': True,
-              'editor/open/browse_workdir': False,
-              'editor/new/browse_scriptdir': False,
-              'editor/new/browse_workdir': True,
-              'editor/open/auto_set_to_basedir': False,
-              'editor/save/auto_set_to_basedir': False,
               'working_dir_adjusttocontents': False,
               'working_dir_history': 20,
-              'startup/use_last_directory': True,
+              'startup/use_project_or_home_directory': True,
+              'console/use_project_or_home_directory': True,
+              'console/use_cwd': False,
+              'console/use_fixed_directory': False,
+              'console/fixed_directory': '',

```

Comment:
I think there's no need for this option because it's used to save user data and so we won't need to change it/touch in the feature. The only options we add to `config/main.py` are the ones we want to change in the future.

So please remove it from here.